src/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveEvaluator.cs

```diff
@@ -283,9 +283,11 @@ public sealed class RInteractiveEvaluator : IInteractiveEvaluator {
             // From R docs:  Valid values are 10...10000 with default normally 80.
             _terminalWidth = Math.Max(10, Math.Min(10000, width));
 
-            Session.OptionsSetWidthAsync(_terminalWidth)
-                .SilenceException<RException>()
-                .DoNotWait();
+            if (Session.IsHostRunning) {
+                Session.OptionsSetWidthAsync(_terminalWidth)
```

Comment:
That think is extension to `IRExpressionEvaluator` so it doesn't have that information. I guess I can just leave this alone